src/avif.c

```diff
@@ -721,6 +721,54 @@ avifBool avifCleanApertureBoxConvertCropRect(avifCleanApertureBox * clap,
     return AVIF_TRUE;
 }
 
+// ---------------------------------------------------------------------------
+
+avifBool avifAreGridDimensionsValid(avifPixelFormat yuvFormat, uint32_t columns, uint32_t rows, uint32_t tileW, uint32_t tileH, avifDiagnostics * diag)
+{
+    // The rules below only apply to grid images. Single-cell images do not have these constraints.
+    if ((columns == 1) && (rows == 1)) {
+        return AVIF_TRUE;
+    }
+
+    // ISO/IEC 23000-22:2019, Section 7.3.11.4.2:
+    //   - the tile_width shall be greater than or equal to 64, and should be a multiple of 64
+    //   - the tile_height shall be greater than or equal to 64, and should be a multiple of 64
+    // The "should" part is ignored here.
+    if ((tileW < 64) || (tileH < 64)) {
+        avifDiagnosticsPrintf(diag,
+                              "[Strict] Grid image tile width (%u) or height (%u) cannot be smaller "
+                              "than 64 if there is more than one cell. "
```

Comment:
I will remove it but why? It is more accurate.

Reply:
The reason is that the libavif decoder calls avifAreGridDimensionsValid() on all image grids, even when the image grid has only one cell. The libavif encoder won't encode an image grid if there is only one cell, but other AVIF encoders may do that because it is allowed by the image format.